personalsite/src/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+ReactDOM.render(document.getElementById('root'));
```

Comment:
`ReactDOM.render(<App/>, document.getElementById('root'));` Where did the `<App/>` go ? :(